osu.Game/Database/OsuDbContext.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using osu.Game.Beatmaps;
+using osu.Game.Input.Bindings;
+using osu.Game.IO;
+using osu.Game.Rulesets;
+
+namespace osu.Game.Database
+{
+    public class OsuDbContext : DbContext
+    {
+        private readonly string connectionString;
+
+        public OsuDbContext()

```

Comment:
what is this? why are there no comments?

Reply:
My bad. Will add comments this evening when I get back home.